lib/tasks/after_tasks.rake

```diff
@@ -433,10 +433,10 @@ namespace :After do
   end
 
 
-  desc "Clean up work URLs for abuse reports from the last month"
-  task(:clean_abuse_report_work_urls => :environment) do
+  desc "Clean up URLs for abuse reports from the last month"
+  task(:clean_abuse_report_urls => :environment) do
```

Comment:
/me butts in

Just once when we deploy -- the new code with automatically tidy up any new reports